job_templates/snippet/property_parameter-definition_common.xml.em

```diff
@@ -63,7 +63,7 @@ choices.remove(ubuntu_distro)
             <a class="string-array">
               <string>@el_release</string>
 @{
-choices = ['9', '8']
+choices = ['9.1', '9', '8']
```

Comment:
```suggestion
choices = ['9.2', '9.1', '9', '8']
```

Is it worth adding 9.2 as a choice now so we can later test it, or should we wait until there's a chance of success?

Reply:
I think it would be fine to have additional minors in the list, but we should probably have a plan for when/how to retire them, since we generally target the 'latest' minor release (except when we get broken like this).